src/source-pcap-file-helper.h

```diff
@@ -121,4 +130,14 @@ TmEcode ValidateLinkType(int datalink, DecoderFunc *decoder);
 
 const char *PcapFileGetFilename(void);
 
+bool ShouldDeletePcapFile(PcapFileFileVars *pfv);
```

Comment:
please use `PcapFile`-prefix here to keep the name space clean (for each of the additions here)